packages/customers/src/CustomerContainer.tsx

```diff
@@ -1,11 +1,36 @@
-import React, { FC } from 'react';
+import { Typography, useHostContext } from '@openmsupply-client/common';
+import React, { FC, useEffect } from 'react';
+import { useMatch } from 'react-router-dom';
 
 const TransactionService = React.lazy(
   () => import('transactions/TransactionService')
 );
 
+const RequsitionsService: React.FC = () => (
+  <Typography style={{ marginLeft: 25, marginTop: 75 }}>
+    coming soon..
+  </Typography>
+);
+
 const CustomerContainer: FC = () => {
-  return <TransactionService />;
+  const { setTitleKey } = useHostContext();
+  const isInvoice = useMatch('/customers/customer-invoice/*');
+  const isRequsition = useMatch('/customers/customer-requisition/*');
+  const titleKey = isInvoice
+    ? 'app.customer_invoices'
+    : isRequsition
+    ? 'app.customer_requisitions'
+    : 'app.customers';
+
+  useEffect(() => setTitleKey(titleKey), [titleKey]);
+  switch (titleKey) {
+    case 'app.customer_invoices':
+      return <TransactionService />;
+    case 'app.customer_requisitions':
+      return <RequsitionsService />;
+    default:
+      return <></>;
+  }
```

Comment:
Should this be using a router? The `TransactionService` has one, which might be a little confusing..

Maybe a router here which can match on the sub menu items i.e. `customer-invoice`, and then rather than a `TransactionService`, there is a `CustomerInvoiceService` which has a router which matches on `/` and `:id`..?

Something for later, I think.. to figure out exactly how the routing should work..

Reply:
😄  Yes, I did this already too!
My problem was that adding a `Router` here affects the routes which `TransactionService` sees. In the way that the route is `/customers/customer-invoice` but is `/customer-invoice` when the `TransactionService` sees it. I also thought that this was not a permanent solution. 

Will have another look ..